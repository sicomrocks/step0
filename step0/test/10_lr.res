HI : 0x00000a45
01 : 0x00000023
16 : 0x000009f4
