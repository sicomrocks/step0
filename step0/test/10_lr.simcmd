#Test fonction lr ; foctionnement normal

lr HI 0xa45
lr $at 0x23
lr $16 0x9f4
