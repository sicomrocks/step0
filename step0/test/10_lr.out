CMD TEST RESULT 34 2629
CMD TEST RESULT 1 35
CMD TEST RESULT 16 2548
